Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM python:3.8-slim AS conpot-builder
+
+# Install dependencies
+#RUN apt update \
+#    && apt install ipmitool git python3-dev build-base py-cffi libxslt-dev libffi-dev py3-pip \
+#    && apt install mysqlclient-dev
+
+RUN apt-get update && apt-get install -y \
+    libmariadb-dev \
+    gcc \
+    && rm -rf /var/lib/apt/lists/*
+
+# Copy the app from the host folder (probably a cloned repo) to the container
+RUN adduser --disabled-password --gecos "" conpot
+
+RUN mkdir /opt/conpot
+COPY . /opt/conpot/
+RUN chown conpot:conpot -R /opt/conpot
+
+# Install Python requirements
+USER conpot
+
+WORKDIR /opt/conpot
+
+ENV PATH=$PATH:/home/conpot/.local/bin
+
+RUN pip3 install --user --no-cache-dir -r requirements.txt
+
+# Install the Conpot application
+RUN python3 setup.py install --user --prefix=
+
+
+# Run container
+FROM python:3.8-slim
+
+RUN apt-get update && apt-get install -y \
+    wget \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN adduser --disabled-password --gecos "" conpot
+WORKDIR /home/conpot
+
+COPY --from=conpot-builder --chown=conpot:conpot /home/conpot/.local/ /home/conpot/.local/
+RUN mkdir -p /etc/conpot /var/log/conpot /usr/share/wireshark \
+    && wget https://github.com/wireshark/wireshark/raw/master/manuf -o /usr/share/wireshark/manuf
```

Comment:
Are `/etc/conpot` or the wireshark stuff used at all?

Reply:
There's no strong need to address this in this particular PR. In fact, I may have just the thing in the pipeline. :smiling_imp:

Reply:
Haha, nice, that would remove a lot of messiness...